beluga_demo_nav2/launch/gazebo_launch.py

```diff
@@ -1,114 +0,0 @@
-# Copyright 2024 Ekumen, Inc.
```

Comment:
@Alondruck note I standardized all demos to use the same simulation infrastructure (which is pretty solid as it is).